packages/ai/src/ai/modules/task/commands/cmd_monitor.py

```
@@ -84,7 +85,15 @@ def __init__(
         # Format: "apikey:token" -> EVENT_TYPE mapping
         self._monitors: Dict[str, EVENT_TYPE] = {}
 
-    async def forward_event(
+    async def send_server_event(self, type: EVENT_TYPE, event: Dict[str, Any]) -> None:
```

Ruff (0.15.7)

[error] Function argument `type` is shadowing a Python builtin

(A002)